trustpinning/certs_test.go

```diff
@@ -183,7 +183,8 @@ func TestValidateRootWithPinnedCert(t *testing.T) {
 	// This call to trustpinning.ValidateRoot should also succeed with the correct Cert ID (same as root public key ID), even though we passed an extra bad one
 	validatedSignedRoot, err = trustpinning.ValidateRoot(nil, &testSignedRoot, "docker.com/notary", trustpinning.TrustPinConfig{Certs: map[string][]string{"docker.com/notary": {rootPubKeyID, "invalidID"}}, DisableTOFU: true})
 	require.NoError(t, err)
-	require.Equal(t, validatedSignedRoot, typedSignedRoot)
+	typedSignedRoot.Signatures[0].IsValid = true
```

Comment:
non-blocking nit: can we add a comment here and to the other `IsValid` assignment below explaining that this extra assignment is necessary because `ValidateRoot` calls through to a successful `VerifySignatures` which marks `IsValid`?